2_adventures/001/jimmylorunning/01_day1/README.md

```diff
@@ -0,0 +1,13 @@
+- `git checkout master`
+- `git remote add upstream https://github.com/RubySteps/21-day-challenge`
+- `git remote -v` (to make sure it's there)
```

Comment:
`git remote show upstream` then to show more details.  Yours is nice for the different information.  An commenting in the requests here is a wonderful way to make notes. :)